api/src/main/java/org/motechproject/nms/api/web/UserController.java

```diff
@@ -38,10 +41,17 @@
 @Controller
 public class UserController extends BaseController {
 
+    public static final String CALLING_NUMBER = "callingNumber";
+    public static final String LANGUAGE_LOCATION_CODE = "languageLocationCode";
+    public static final String SERVICE_NAME = "serviceName";
```

Comment:
I just did that to get checkstyle to shut up.  @fhuster gets testy when I
check in code that doesn't compile.

These are API level constants, so I think it might make sense to extract
all the json keys and put them in an api/json-keys.java file or something
like it.

On Thu, Apr 23, 2015 at 9:58 AM, kosh <notifications@github.com> wrote:

> In api/src/main/java/org/motechproject/nms/api/web/UserController.java
> <https://github.com/koshalt/mim/pull/74#discussion_r28981429>:
>
> > @@ -38,10 +41,17 @@
> >  @Controller
> >  public class UserController extends BaseController {
> >
> > +    public static final String CALLING_NUMBER = "callingNumber";
> > +    public static final String LANGUAGE_LOCATION_CODE = "languageLocationCode";
> > +    public static final String SERVICE_NAME = "serviceName";
>
> Are these the type of constants that we envision putting in the
> config/props package or is that just magic numbers and such?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/74/files#r28981429>.
>